daisy/step_wait_for_instances_signal.go

```diff
@@ -71,6 +73,16 @@ type SerialOutput struct {
 	StatusMatch  string         `json:",omitempty"`
 }
 
+// GuestAttribute describes text signal strings that will be written to guest
+// attributes.
+// This step will not complete until the key exists and matches the value in
+// SuccessValue (if specified and non empty). If SuccessValue is set, any other
+// value in the key will cause the step to fail.
+type GuestAttribute struct {
+	KeyName      string `json:",omitempty"`
+	SuccessValue string `json:",omitempty"`
+}
+
```

Comment:
Unit tests?

Also, a test similar to: daisy_integration_tests/step_wait_for_instance_signal.wf.json

Reply:
added unit and integ tests